src/components/AddObsBottomSheet/AddObsBottomSheet.tsx

```diff
@@ -0,0 +1,169 @@
+      testID: "camera-button",
+      accessibilityLabel: t( "Camera" ),
+      accessibilityHint: t( "Navigates-to-camera" )
+    },
+    photoLibrary: {
+      text: t( "Upload-photos" ),
+      icon: "photo-library",
+      onPress: ( ) => navAndCloseBottomSheet( "PhotoLibrary" ),
+      testID: "import-media-button",
+      accessibilityLabel: t( "Photo-importer" ),
+      accessibilityHint: t( "Navigates-to-photo-importer" )
+    },
+    soundRecorder: {
+      text: t( "Record-a-sound" ),
+      icon: "microphone",
+      onPress: ( ) => navAndCloseBottomSheet( "SoundRecorder" ),
+      testID: "record-sound-button",
+      accessibilityLabel: t( "Sound-recorder" ),
+      accessibilityHint: t( "Navigates-to-sound-recorder" )
+    },
+    noEvidence: {
+      text: t( "Create-observation-with-no-evidence" ),
+      icon: "noevidence",
+      onPress: async ( ) => {
+        const newObservation = await Observation.new( );
+        prepareObsEdit( newObservation );
+        navAndCloseBottomSheet( "ObsEdit" );
+      },
+      testID: "observe-without-evidence-button",
+      accessibilityLabel: t( "Observation-with-no-evidence" ),
+      accessibilityHint: t( "Navigates-to-observation-edit-screen" )
+    }
+  } ), [
+    navAndCloseBottomSheet,
+    prepareObsEdit,
+    t
+  ] );
+
+  const renderAddObsIcon = ( {
+    accessibilityHint,
+    accessibilityLabel,
+    icon,
+    onPress,
+    testID,
+    text
+  }: ObsCreateItem ) => (
+    <Pressable
+      className="bg-white w-1/2 flex-column items-center py-4 rounded-lg flex-1 shadow-sm
+      shadow-black/25 active:opacity-50"
+      onPress={onPress}
+      accessibilityHint={accessibilityHint}
+      accessibilityLabel={accessibilityLabel}
+      testID={testID}
+    >
+      <INatIconButton
+        className={GREEN_CIRCLE_CLASS}
+        accessibilityHint={accessibilityHint}
+        accessibilityLabel={accessibilityLabel}
+        color={String( colors?.white )}
+        icon={icon}
+        onPress={onPress}
+        size={icon === "aicamera"
+          ? 28
+          : 20}
+      />
+      <Body3>{text}</Body3>
+    </Pressable>
+  );
+
+  return (
+    <BottomSheet
+      onPressClose={closeBottomSheet}
+      hidden={hidden}
+      insideModal={false}
+      hideCloseButton
+      additionalClasses="bg-lightGray pt-4"
+      scrollEnabled={false}
+    >
+      <View className="flex-column gap-y-4 pb-4 px-4">
+
+        <View className={ROW_CLASS}>
+          {renderAddObsIcon( obsCreateItems.standardCamera )}
+          {renderAddObsIcon( obsCreateItems.photoLibrary )}
+        </View>
+
+        <View className={ROW_CLASS}>
+          {renderAddObsIcon( obsCreateItems.soundRecorder )}
+          {AI_CAMERA_SUPPORTED && renderAddObsIcon( obsCreateItems.aiCamera )}
+        </View>
+
+        <Pressable
+          className="bg-mediumGray w-full flex-row items-center py-[10px] px-5 rounded-lg
+          active:opacity-50"
```

Comment:
How did you land on `-50`? Is there a paper trail to that decision? Is that something we should add to `Pressable` coming from `styledComponents` so that all our Pressables behave the same way?

Not in this PR though.